types/src/multi_signature.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_crypto::bls12381;
+use aptos_crypto_derive::{BCSCryptoHash, CryptoHasher};
+use std::collections::HashMap;
+
+use move_deps::move_core_types::account_address::AccountAddress;
+use serde::{Deserialize, Serialize};
+
+/// This struct represents the aggregated BLS signature representation that contains an aggregated
+/// BLS signature and a bit mask representing the set of validators participating in the signing
+/// process
```

Comment:
I'd rephrase as:

 > This struct represents a BLS multisignature: it stores a bit mask representing the set of validators participating in the signing process and the multisignature itself, which was aggregated from these validators' partial BLS signatures.